packages/flutter/test/material/floating_action_button_location_test.dart

```diff
@@ -316,6 +316,28 @@ void main() {
     expect(tester.getCenter(find.byType(FloatingActionButton)).dy, kToolbarHeight);
   }, skip: isBrowser);
 
+  testWidgets('Mini-start-top floating action button location, with custom toolbarHeight', (WidgetTester tester) async {
+    const double customToolbarHeight = 128.0;
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          appBar: AppBar(toolbarHeight: customToolbarHeight),
+          floatingActionButton: FloatingActionButton(onPressed: () { }, mini: true),
+          floatingActionButtonLocation: FloatingActionButtonLocation.miniStartTop,
+          body: Column(
+            children: const <Widget>[
+              ListTile(
+                leading: CircleAvatar(),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+    expect(tester.getCenter(find.byType(FloatingActionButton)).dx, tester.getCenter(find.byType(CircleAvatar)).dx);
+    expect(tester.getCenter(find.byType(FloatingActionButton)).dy, customToolbarHeight);
+  }, skip: isBrowser);
```

Comment:
Why is this test being skipped? We should include a link to an issue that is causing this to be skipped.
```suggestion
  }, skip: isBrowser); // link to issue
```

Reply:
Ah, it looks like a bunch of tests are being skipped here. I think this can go without the skip, I did a quick check and it looks like these do not need to be skipped anymore, see #54322 
```suggestion
  });
```